examples/js/loaders/GLTFLoader.js

```diff
@@ -1421,6 +1471,18 @@ THREE.GLTFLoader = ( function () {
 
 	}
 
+	GLTFParser.prototype.setExtensions = function ( extensions ) {
+
+		this.extensions = extensions;
+
+	};
+
+	GLTFParser.prototype.setPlugins = function ( plugins ) {
+
+		this.plugins = plugins;
+
+	};
```

Comment:
We don't intend for users to call setPlugins or setExtensions, right?

Reply:
Right, I don't expect users to call them.